src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="ul">
+          <li class="nav-li">
+            <a
+              href="#Apple"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. The href attribute for 'Apple' contains an uppercase letter and a space which may cause issues in URL encoding. Use lowercase and hyphens instead (e.g., '#apple').